prerequisites.rst

```diff
@@ -1,35 +1,250 @@
-Prerequisites
+Requisites
 =============
 
-Before installing |omv| make sure your hardware is supported.
+The |omv| system is the most flexible Network Attached Storage (NAS) on the market,
+this is because it relies on the operating system Debian GNU/Linux [9]_
 
-* **CPU**: Any x86-64 or ARM compatible processor
-* **RAM**: 1 GiB capacity
-* **HDD**:
+Any system has software requirements and hardware requirements, also minimal
+ones and recommended ones..
 
-  * **System Drive**: min. 4 GiB capacity (plus the capacity of the RAM)
-  * **Data Drive**: capacity according to your needs
+Hardware requirements
+---------------------
 
-.. note::
-   The whole disc will be occupied by the system and swap space [1]_, so size
-   doesn't matter so much. Data storage on the system disc is not supported.
+======  ================  =========  ==========  ==================================
+ Item    supported         Minimal     Best       Recommendation
+======  ================  =========  ==========  ==================================
+ DRIV    SSD/HDD/USB...    1 disk      2 disk     2 disks: 128Gib HDD + 500Gib SSD
+ RAM     1Gib+ any          1Gib       4Gig       8Gib+ dual channel DDR4/DDR3
+ NIC     WiFi/Ether/USB     any       10Mb NIC    1GiB NIC or 10Gb NICs: SFP fiber
+ CPU     arm,x86,x64        32bit      64bit      Intel Dual Core, AMD Ryzen
+======  ================  =========  ==========  ==================================
 
-.. note::
-   Take into account in your considerations about the size of the system disk
-   that various plugins such as `openmediavault-podman` or `openmediavault-k8s`
-   store their data (e.g. container images) on the system disk.
+The storage drives (DRIV)
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Spinning Harddisk, SSD [2]_, Disk-on-Module [3]_, CompactFlash [4]_ or USB thumb
-drive [5]_ type drives can be used as system drive.
+This is the key of the system. |omv| supports any drive hardware and any drive 
+interface (SATA, ISE, SCSI, USB, SERIAL) but information and management will
+depend on the disk drives supported by the Debian operating system.
+
+Modern hard disks drives have firmware inside that reports several attributes, 
```

Comment:
```suggestion
Modern hard disks drives have firmware inside that reports several attributes.
```

Reply:
fixed in commit be624bd8da52ad54039c7533a818c8b7073b30b2